src/FileUtils/__test__/AgGridUtils.spec.js

```diff
@@ -140,3 +140,77 @@ describe('export to CSV string', () => {
     expect(res).toStrictEqual(expectedOutput);
   });
 });
+
+describe('force all columns to have a headerName', () => {
+  const columns = [{ field: 'field' }];
+  const columnsWithSomeHeaderNames = [
+    { field: 'firstField' },
+    { field: 'secondField' },
+    { field: 'thirdField', headerName: 'Third field ' },
+  ];
+  const columnsWithGroups = [
+    { field: 'firstField' },
+    { headerName: 'firstGroup', children: [{ field: 'secondField' }, { field: 'thirdField' }] },
+    {
+      headerName: 'firstGroup',
+      children: [
+        { field: 'secondField' },
+        { headerName: 'secondGroup', children: [{ field: 'fourthField' }, { field: 'fifthField' }] },
+      ],
+    },
+    { field: 'sixthField', headerName: 'sixthField' },
+  ];
+  const columnsWithGroupsWithHeaders = [
+    { field: 'firstField', headerName: 'firstField' },
+    {
+      headerName: 'firstGroup',
+      children: [
+        { field: 'secondField', headerName: 'secondField' },
+        { field: 'thirdField', headerName: 'thirdField' },
+      ],
+    },
+    {
+      headerName: 'firstGroup',
+      children: [
+        { field: 'secondField', headerName: 'secondField' },
+        {
+          headerName: 'secondGroup',
+          children: [
+            { field: 'fourthField', headerName: 'fourthField' },
+            { field: 'fifthField', headerName: 'fifthField' },
+          ],
+        },
+      ],
+    },
+    { field: 'sixthField', headerName: 'sixthField' },
+  ];
+
+  const columnsWithHeaderNames = [
+    { field: 'field', headerName: 'First field' },
+    { field: 'secondField', headerName: 'Second field' },
+  ];
+  test.each`
+    columns      | result
+    ${[]}        | ${[]}
+    ${{}}        | ${[]}
+    ${null}      | ${[]}
+    ${undefined} | ${[]}
+    ${''}        | ${[]}
+  `('empty array is returned is columns list is invalid', ({ columns, result }) => {
```

Comment:
Typo: `is columns list is invalid` -> `if columns list is invalid`

Reply:
Nice catch, thanks ! :pray: